File: Java/1486.XOROperationinanArray.java
class Solution {
    public int xorOperation(int n, int start) {
        int nums[]=new int[n];
        int i,bit=0;
        for(i=0;i<nums.length;i++) nums[i]=start+2*i;
        for(i=0;i<nums.length;i++) bit^=nums[i];
        return bit;
    }
}

File: Python/1486.XOROperationInAnArray.py
class Solution:
    def xorOperation(self, n: int, start: int) -> int:
        nums = []
        for i in range(n):
            suma = start+2*i
            nums.append(suma)
        suma = 0
        for k in range(n):
            suma = suma^nums[k]
        return suma
        
